src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -216,13 +216,7 @@ def put(
         is_update_org_role = assigned_org_role and assigned_org_role != member.role
 
         if is_update_org_role:
-            if getattr(member.flags, "idp:role-restricted"):
-                return Response(
-                    {
-                        "role": "This user's org-role is managed through your organization's identity provider."
-                    },
-                    status=403,
-                )
+            # TODO(adas): Reenable idp lockout once all scim role bugs are resolved.
```

Comment:
Should we comment it out instead of deleting it?

Reply:
Leaving commented code lying around isn't good code hygiene. It's better to save the PR and revert.